test/e2e/framework/unmanaged.go

```diff
@@ -152,15 +159,29 @@ func (f *UnmanagedFramework) ClusterClients(userAgent string) map[string]kubecli
 
 	kubeClient := f.KubeClient(userAgent)
 	crClient := f.CrClient(userAgent)
-	clusterClients := make(map[string]kubeclientset.Interface)
+	testClusters := make(map[string]common.TestCluster)
 	for _, cluster := range clusterList.Items {
 		ClusterIsReadyOrFail(fedClient, cluster)
 		config, err := util.BuildClusterConfig(&cluster, kubeClient, crClient)
 		Expect(err).NotTo(HaveOccurred())
 		restclient.AddUserAgent(config, userAgent)
-		clusterClients[cluster.Name] = kubeclientset.NewForConfigOrDie(config)
+
+		// Check if this cluster is the same name as the current-context to
```

Comment:
Is this the best way to do this in the absence of a first-class cluster id?  I'm wondering if it might make sense to require a configuration parameter naming the host cluster instead.